app/src/main/java/ch/epfl/sdp/artificial_intelligence/MovingArtificialEntity.java

```diff
@@ -7,25 +7,19 @@
 
 public class MovingArtificialEntity extends MovingEntity implements Movable, Localizable, Updatable {
     private GenPoint position;
-    private float velocity;
-    private float acceleration;
-    private double orientation;
-    private Movement movement;
     private boolean moving;
     private Boundable bounds;
-    public double sinusAmplitude = 1;
-    public double sinusAngle;
+    private Movement movement;
+    private boolean isActive;
```

Comment:
I would rather call it isMoving, what do you think?

Reply:
This symbolize the fact that the entity is still displayed on the map or not (it means that if this Boolean is set to false, we undisplay the entity and we remove it from the displayable list ). So isMoving is also a bad choice, but yeah, we would need to change this